### PR TITLE
feat: bump dependencies version

### PR DESCRIPTION
**Pull request by bzp2010**
Upgrade dependencies with upstream Ant Design Pro project.

### CODE REVIEW

**Comment by juzhiyuan** [comment]
from V5?

**Comment by bzp2010** [comment]
> from V5?

Yes, It come from V5 branch.

File: package.json [hunk 1]
```diff
@@ -44,10 +44,15 @@
...
+  "browserslist": [
+    "> 1%",
+    "last 2 versions",
+    "not ie <= 10"
+  ],
   "dependencies": {
-    "@ant-design/icons": "^4.2.1",
-    "@ant-design/pro-layout": "6.0.0-2",
-    "@ant-design/pro-table": "^2.3.3",
+    "@ant-design/icons": "^4.0.0",
```
Comment (juzhiyuan) [inline package.json:53]: I'm not sure if we should use some older versions...

Comment (bzp2010) [reply package.json:53]: `^` Should mean the maximum version greater than 4.0.0.

In [yarn.lock](https://github.com/apache/incubator-apisix-dashboard/blob/af555bc839759dca1b5268ecf523e789436a1fbc/yarn.lock#L30) can prove it.

**Pull request merged**